README.md

```diff
@@ -220,6 +234,83 @@ charts:
           - linux/arm64
 ```
 
+### Controlling development versions in Chart.yaml
+
+Like some "package version in version control" tools,
+you don't need to manage versions at all in chartpress except by tagging commits.
+
+However, relying only on tags results in "development versions" (versions published from commits after a release)
+having somewhat confusing prerelease versions.
+
+After publishing e.g. `1.2.3`, the next version will be `1.2.3-0.dev.git.1.habc`.
+According to Semantic Versioning,
+this is a "pre release" (good, it should be excluded from default installation),
+but it means it comes _before_ 1.2.3 (wrong! it's 1 commit _newer_ than 1.2.3).
+This is because prereleases should be defined relative to the _next_ release,
+not the _last_ release. But git tags only store the _last_ release!
+
+Chartpress 2.0 adds an option `chart.useChartVersion`,
```

Comment:
Sorry, I don't know what you mean. What do you think the version should be?

Reply:
If this is a new feature without a breaking change that you opt-in to, then i figure we would make a minor version bump rather than a major version bump!

Reply:
Got it, sorry. Since this does change the tag format (`-n005.habc` -> `-0.dev.git.5.habc`). I don't _think_ that'll matter to most folks, but it may enough that a major version feels appropriate.